src/applications/static-pages/analytics-opt-out/app.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import recordEvent from 'platform/monitoring/record-event';
+
+const AnalyticsOptOut = () => {
+  const gaProperty = 'UA-XXXX-Y';
+  const disableStr = `ga-disable-${gaProperty}`;
+  const [optedOut, setOptedOut] = useState(
+    document.cookie.indexOf(`${disableStr}=true`) > -1,
+  );
+
+  const gaOptOut = () => {
+    // set cookie, and window variable as per google documentation, set optedOut state to true
+    document.cookie = `${disableStr}=true;`;
+    window[disableStr] = true;
+    setOptedOut(true);
+  };
+
+  const handleOptOut = () => {
+    // record event and run opt out
+    recordEvent({ event: 'analytics-opt-out', 'internal-user': 'true' });
+    gaOptOut();
+  };
+
+  return (
+    <div>
+      <div className="main home" role="main">
+        <div className="section main-menu">
+          <div className="row">
+            <div className="small-12 columns">
+              <div style={{ padding: '2em 0' }}>
```

Comment:
Can we use design system classes here? `vads-u-padding-y--4` https://design.va.gov/utilities/padding

Reply:
I was thinking the same thing, I just copied the code from the original fil. Ill update it now to use the class instead